tests/test_ext_marshmallow.py

```diff
@@ -598,7 +602,8 @@ def test_schema_array_uses_ref_if_available_v3(self, spec_fixture):
                 "get": {
                     "parameters": [
                         {
-                            "in": "body",
+                            "name": "Pet",
+                            "in": "query",
```

Comment:
Yes, since we now require "name" key in parameters test fail and raises an `InvalidParameterError`

Reply:
I think I wrote this comment before you introduced `InvalidParameterError` because I didn't see where the failure would actually happen. Now, the reason for the change is clear.